app/models/services/locators/pubmed_paper_locator.rb

```diff
@@ -23,7 +24,10 @@ def find_or_import_paper
   end
 
   def valid?
-    only_numbers = /^[0-9]+$/
-    return locator.match(only_numbers)
+    only_numbers = !!locator_id.match(/^[0-9]+$/)
+
+    errors << "\"#{locator_id}\" does not match Pubmed ID format. Ex: \"18365029\"" unless only_numbers
```

Comment:
something else to consider in the future.  There's the pubmed database and then there's also PMC which has IDs like PMC3541491.  If I understand it correctly PMC has full text articles:

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3541491/

I'm not 100% sure how it ties to pubmed bibliographic data yet, but I'll keep researching.  I should prob also reach out to pubmed as well and see if we can get a learning session or Q&A session as some point.